docs/dev/clusteroperator.md

```diff
@@ -85,7 +91,7 @@ If the controller reaches 4.0.1, the conditions might be:
 If an error blocks reaching 4.0.1, the conditions might be:
 
 * `Failing` is true with a detailed message `Unable to apply 4.0.1: could not update 0000_70_network_deployment.yaml because the resource type NetworkConfig has not been installed on the server.`
-* `Available` is false with message `Cluster is unable to reach 4.0.1`
+* `Available` is true with message `Cluster has deployed 4.0.0`
```

Comment:
Of course, now that I really read through this, if NetworkConfig doesn't exist as a CRD we should be paging someone.